ambry-server/src/main/java/com/github/ambry/server/AmbryServerSecurityService.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.server;
+
+import com.github.ambry.commons.ServerMetrics;
+import com.github.ambry.config.ServerConfig;
+import com.github.ambry.rest.RestRequest;
+import com.github.ambry.rest.RestServiceErrorCode;
+import com.github.ambry.rest.RestServiceException;
+import com.github.ambry.router.Callback;
+import io.netty.channel.ChannelHandlerContext;
+
+/**
+ * Default implementation of {@link ServerSecurityService} for Ambry that doesn't do any validations.
+ */
+public class AmbryServerSecurityService implements ServerSecurityService {
```

Comment:
How do you think of moving this class to `com.github.ambry.rest` ? so that frontend can also leverage it if needed.

Reply:
frontend already has a SecurityService interface/class which does the security validations. This is defined in ambry-api and implmented in ambry-frontend. I mainly followed the same pattern from this frontend class to apply for server.